web/packages/core/src/ruffle-player.ts

```diff
@@ -1127,7 +1127,7 @@ export class RufflePlayer extends HTMLElement {
         const issueTitle = `Error on ${pageUrl}`;
         let issueLink = `https://github.com/ruffle-rs/ruffle/issues/new?title=${encodeURIComponent(
             issueTitle
-        )}&labels=error-report&body=`;
+        )}&template=error_report.md&labels=error-report&body=`;
```

Comment:
Isn't the `labels=error-report` redundant given `template=error_report.md`?

Reply:
From my testing both are required to work in all cases. Confusingly it works differently depending on if the user has write access or not -- i.e. if a general user submits an error report, template=error_report.md is enough, but if I submit one, it needs labels=error-report because I have the access to set a label.

Reply:
The alternative is we could use a bot to auto-label all issues with "Error on".

Reply:
Seems good enough to stay like this.